neo4j/neo4j.adoc

```diff
@@ -0,0 +1,28 @@
+= How To Use Neo4j
+:section: Neo4j
+:section-link: neo4j
+:section-level: 1
+:slug: neo4j
+
+= image:{img}icon-guide-ecosystem.png[] How To Use Neo4j
+
+There are a variety of ways to interact and use Neo4j.
+This guide introduces some of the different products at Neo4j that suit different business and technology needs and how to get started using them.
+
+From our core graph database to visualization for business users, we have a variety of opportunities to maximize your business and simplify your data.
+
+
+== What Does Neo4j Offer?
+
+Each of the products listed below is covered in more detail in subsequent pages.
+We continue to improve our product list, as well as capabilities of the products themselves.
+If you have any questions or issues, don't hesitate to http://neo4j.com/contact-us/[contact our team^]!
+
```

Comment:
Putting these 5 things in a list is a bit awkward because the you use the **browser** from the **desktop** to manage the **database**. 

I don't know if you should change anything, just a thought :)